### ISSUE TITLE
[Question] Is there a map editor using pixi-tilemap?

### ISSUE THREAD

**Issue by Soliaonline** [pixijs/tilemap#57]
Is there a javascript based map editor for making Maps instead of using RPG maker itself?

**Comment by ivanpopelyshev** [comment]
Nope. pixi-tilemap provides a way to show tilemap independently of the editor you choose. Rpgmaker maps work through rpgmaker runtime that uses pixi-tilemap.

**Comment by blurymind** [comment]
@Soliaonline this was actually discussed on the gdevelop trello board. If you want to build one, you are not alone :)  We have the same goal! So why not come by and check our repo here:
https://github.com/4ian/GDevelop
try gdevelop5 :) we use pixijs under the hood for both the electron editor and the game engine for rendering.

**Comment by Soliaonline** [comment]
> 
> 
> @Soliaonline this was actually discussed on the gdevelop trello board. If you want to build one, you are not alone :) We have the same goal! So why not come by and check our repo here:
> https://github.com/4ian/GDevelop
> try gdevelop5 :) we use pixijs under the hood for both the electron editor and the game engine for rendering.

I built one a while back, but I am trying to optimize things further.  Right now I am working on an Atlas Creator/Editor.  It's not amazing or anything, but it accomplish what I am after.   I feel the way I need it to work is not beneficial for a community because I am not handling anything, but the floor tiles.  No buildings, no trees, only tiles.  My goal is to make something akin to the old starcraft map editor.   I did download and checkout RPG Maker MV to see how they solved problems, and realized that they did things much the same way I did with what I call "default" tile state.   Where everything is set to have dirt around it if it it outside of it's element.
